app/controllers/stories_controller.rb

```diff
@@ -460,4 +462,31 @@ def track_story_reads
     yield
     @ribbon.touch
   end
+
+  def repost_story_description
+    if ActiveRecord::Type::Boolean.new.deserialize(story_params[:repost_description])
+      @story.description= ""
```

Comment:
```suggestion
      @story.description = ""
```

Reply:
Please also set `@story.editor = @user.id`, then you can remove the `try` below on line 484. If a moderator editing a story doesn't create an entry in the moderation log, we should error out.